Assets/Prefabs/Atom/Packs/Boron.prefab

```diff
@@ -127,7 +127,7 @@ MonoBehaviour:
   protons: 5
   neutrons: 5
   electrons: 5
-  atomName: Boron
+  atomName: Borium
```

Comment:
This should be "Boor"

Reply:
Boor en Borium kan allebei :)
alle andere dingen eindigen met ium dus vandaar de keuze